tools/windows_build.bat

```diff
@@ -48,4 +48,8 @@ commonUnitTests.exe -xunitxml >> tests\rmsUnitTestResults.txt
 if %ERRORLEVEL% NEQ 0 ( 
 	exit /B 1
 )
+zipFileUnitTests.exe -xunitxml >> tests\rmsUnitTestResults.txt
```

Comment:
>sts.exe -xunitxml >> tests\rmsUnit [](start = 13, length = 34)

Your additions in this script getting the build script to be stuck

Reply:
I see,
waiting to Shenhav to be able to enter the build machine and see what I can understand from there.

---
In reply to: [139593903](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139593903) [](ancestors = 139593903)